### DISCUSSION TITLE
How can we validate the changes we are doing will work or fail ?

### DISCUSSION THREAD

**Discussion by rajat053117** [renovatebot/renovate#28060] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

Hi Team , 
we are using a centralized repo named as renovate from where renovate runs , it has a renovate-global.json file and a renovate.json file which manages the github actions dependencies 
We run renovate on other repos with auto discover filter for those repos , and each repo has its own individual renovate.json file to manage its dependencies 
What i am looking for is suppose i am making change in any renovate.json file in any repo , will it be possible to run any tests which will shows the change before merging the PR . 
Thanks in advance for the Help 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
https://docs.renovatebot.com/config-validation/
